torch/_torch_docs.py

```diff
@@ -2701,6 +2701,43 @@ def merge_dicts(*dicts):
     input (Tensor): the PyTorch tensor to test
 """)
 
+add_docstr(torch.is_nonzero,
+           r"""
+is_nonzero(input) -> (bool)
+
+Returns True if the :attr:`input` is a scalar which is not equal to zero
+after type conversions.
+i.e. not equal to ``torch.Tensor([0.])`` or ``torch.Tensor([0])`` or
+``torch.Tensor([False])``.
+Throws a ``RuntimeError`` if ``torch.numel() != 1`` (after coalescing in case
+of sparse tensors).
+
+Args:
+    input (Tensor): the PyTorch tensor to test
+Example::
+
+    >>> torch.is_nonzero(torch.Tensor([0.]))
+    False
+    >>> torch.is_nonzero(torch.Tensor([1.5]))
+    True
+    >>> torch.is_nonzero(torch.Tensor([False]))
+    False
+    >>> torch.is_nonzero(torch.Tensor([True]))
+    True
+    >>> torch.is_nonzero(torch.Tensor([0]))
+    False
+    >>> torch.is_nonzero(torch.Tensor([3]))
+    True
+    >>> torch.is_nonzero(torch.Tensor([]))
+    Traceback (most recent call last):
+    ...
+    RuntimeError: bool value of Tensor with no values is ambiguous
+    >>> torch.is_nonzero(torch.sparse_coo_tensor([0]))
```

Comment:
Probably don't need this sparse example.

Reply:
Removed